### DISCUSSION TITLE
AttributeError: Can't get attribute 'DataEdgeAttr'

### DISCUSSION THREAD

**Discussion by black-angela** [pyg-team/pytorch_geometric#7411] [Q&A]
self.training_graphs = GEDDataset('datasets/{}'.format(self.config.dataset), self.config.dataset, train=True)
encountered an error about AttributeError: Can't get attribute 'DataEdgeAttr'.how to solve such a problem

**Comment by rusty1s** [comment]
What do I need to reproduce this issue? This works fine for me:
```python
from torch_geometric.datasets import GEDDataset

dataset = GEDDataset(root='/tmp/ged', name='AIDS700nef')
print(dataset)
```
Can you do me a favor and remove any processed data from disk and try again?
